.github/workflows/testing.yml

```diff
@@ -173,50 +200,50 @@ jobs:
       - name: Integration Tests
         run: go test -v -timeout 30m -parallel 10 -coverpkg=./... -coverprofile=./cover.out.tmp ./...
         env:
-          DYNAMO_ACCESS_KEY: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          DYNAMO_SECRET_KEY: ${{ secrets.AWS_SECRET_KEY }}
-          DYNAMODB_REGION: "us-east-1"
+          # DYNAMO_ACCESS_KEY: ${{ secrets.AWS_ACCESS_KEY_ID }}
```

Comment:
Do we want to leave these commented out or simply delete them?

Reply:
I will remove them once I get all the tests working on CICD and all the environment variables passed through.